stellarphot/core.py

```diff
@@ -1030,6 +1032,11 @@ def from_vizier(
             If True, the catalog data does not contain error information, so a
             column of NaNs will be added for the error values.
 
+        tidy_catalog : bool, optional
+            If True, the catalog will be tidied into a long format with one
```

Comment:
I must admit I am not sure what you mean by 'tidied'.... can you possibly document this a bit more clearly, explaining that it reformats the table keeping magnitudes and colors more cleanly?

Reply:
Done